src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/RazorSemanticTokensInfoService.cs

```diff
@@ -192,7 +310,7 @@ private static void AddAdditionalCSharpWhitespaceRanges(List<SemanticRange> razo
                 Start = new Position(originalRange.Start.Line, previousRazorSemanticRange.End.Character),
                 End = originalRange.Start
             };
-            razorRanges.Add(new SemanticRange(textClassification, whitespaceRange, (int)RazorSemanticTokensLegend.RazorTokenModifiers.RazorCode, fromRazor: false));
+            razorRanges.Add(new SemanticRange(textClassification, whitespaceRange, (int)RazorSemanticTokensLegend.RazorTokenModifiers.RazorCode, fromRazor:false));
```

Comment:
I'm curious: If you run the Format Document command, does this become `fromRazor: false` (with a space between the colon and the `false`)? On my machine it would, so if its not happening on yours, it indicates we have something missing from our `.editorconfig` file.